src/modules/musig/session_impl.h

```diff
@@ -74,12 +74,24 @@ static int secp256k1_musig_pubnonce_load(const secp256k1_context* ctx, secp256k1
     return 1;
 }
 
+static const unsigned char secp256k1_musig_aggnonce_magic[4] = { 0xa8, 0xb7, 0xe4, 0x67 };
```

Comment:
`SECP256K1_SCHNORRSIG_EXTRAPARAMS_MAGIC` is special because the user of the API needs to know it. `secp256k1_musig_aggnonce_magic` on the other hand is internal.